pom.xml

```diff
@@ -234,7 +234,7 @@
         <dependency>
             <groupId>software.amazon.awssdk.iotdevicesdk</groupId>
             <artifactId>aws-iot-device-sdk</artifactId>
-            <version>1.18.1</version>
+            <version>1.19.1</version>
```

Comment:
this should be pulling your custom version shouldn't it?

Reply:
My plan is to: 

1. Get this change onto main because it's separate from the feature work.
2. Build up my changes, including my snapshot version, on a feature branch (built on top of this).
3. Once we're ready to merge the feature to main, I'll submit PRs to both the smithy package and nucleus main branches.

I'm open to other approaches, let me know if you'd recommend something else

Reply:
Sure that's fine